azure-servicebus/src/main/java/com/microsoft/azure/servicebus/ClientSettings.java

```diff
@@ -72,4 +77,56 @@ public Duration getOperationTimeout()
     }
 
     public TransportType getTransportType() { return transportType; }
```

Comment:
>public TransportType getTransportType() { return transportType; } [](start = 4, length = 65)

ah, we missing a comment here.

Reply:
Should be fixed in the new commit, as should the failing test